### PR TITLE
TME-117 - New RAG on how to extend CF access native features with ser…

### PR DESCRIPTION
**Pull request by nouvellonsteph**
…verless computing

### Summary

New RAG demonstration how Cloudflare Access native capabilities can be expanded with serverlesss computing

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

**Review by securitypedant — APPROVED** [review]
Awesome new reference architecture

**Comment by bllchmbrs** [comment]
Hey @nouvellonsteph , Hyperlint developer here. Something unexpected is going on with this PR. I'm looking at it now and will edit this comment with more information.

---
Edit 1: The outstanding issue that Hyperlint has on this PR is a `Terms-error`. It can safely be ignored as it refers to a pedantic issue about `JSON` being capitalized in the URL, based on your URL configuration.

I am working on resolving this issue now to get you to ✅

---
Edit 2: would you mind emailing support@hyperlint.com I have a small follow up question.

File: src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  "iss": "https://myorg.cloudflareaccess.com",
+  "type": "app",
+  "identity_nonce": "IA0hPRvwILtbUXSQ",
+  "sub": "ce40d564-c72f-475f-a9b8-f395f19ad986",
+  "device_id": "8469d7c4-83a9-11ee-b559-76e6e80876db",
+  "country": "FR"
+}
+```
+
+Cloudflare exposes a specific [endpoint](/cloudflare-one/identity/authorization-cookie/validating-json/#_top) to allow anyone to validate and expand a Cloudflare signed JWT.
```
[the remark this answers is not in the document]
Comment (bllchmbrs) [reply src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx:90]: @nouvellonsteph , resolving this comment will give you the ✅!

As mentioned above, it'd be helpful to email support @ hyperlint. I've got a quick question or two that's just not relevant for this PR.

Please ping me directly on GH with any other questions or concerns!

File: src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  order: 1
+  label: ZTNA with external authorization
+updated: 2024-10-10
+---
+
+import { RelatedProduct, LinkCard} from "~/components";
+
+## Introduction
+
+Companies using Zero Trust Network Access (ZTNA) services build policies to determine if a user can access a protected resource such as a privately hosted Wiki server or source code repository. Policies typically use group membership, authentication methods, device security posture to determine which users can access which resources.
```
Comment (haleycode) [inline src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx:17]:
```suggestion
Companies use Zero Trust Network Access (ZTNA) services to build policies to determine if a user can access protected resources such as privately hosted wiki servers or source code repositories. Policies typically use group membership, authentication methods or device security posture to determine which users can access which resources. 
```

File: src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+Companies using Zero Trust Network Access (ZTNA) services build policies to determine if a user can access a protected resource such as a privately hosted Wiki server or source code repository. Policies typically use group membership, authentication methods, device security posture to determine which users can access which resources. 
+
+Secure access requires a range of attributes being available to the policy engine for evaluation. With Cloudflare's ZTNA service, [Access](/cloudflare-one/policies/access/), it is possible to include in the policy an external request to another API that provides part of the data required for the access decision. 
+
+For example, you might have a policy which states all members of the group "Engineers", who have authenticated with credentials that required a hard token, can have access to the self-hosted source code repository. But you also want to only allow engineers who have completed security training. That data might be available in another system, so Cloudflare allows you to, as part of the policy check, make a call using [Workers](https://workers.cloudflare.com/) to the training system to determine if this user has passed security training.
```
Comment (haleycode) [inline src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx:21]:
```suggestion
For example, you might have a policy which states all members of the group "Engineers", who have authenticated with credentials that required a hard token, can have access to the self-hosted source code repository. But you also want to only allow engineers who have completed security training. That data might be available in another system, so as part of the policy check, Cloudflare allows you to make a call to the training system using [Workers](https://workers.cloudflare.com/) to determine if a user has passed security training.
```

File: src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+This document outlines how to combine both solutions to enhance Cloudflare Access capabilities in terms of [authorization and authentication](https://www.cloudflare.com/learning/access-management/what-is-access-control/).
+
+## Showcased products
+
+<RelatedProduct header="Workers" href="/workers/" product="workers">
+    Build serverless applications and deploy instantly across the globe for exceptional performance, reliability, and scale.
+</RelatedProduct>
+
+<RelatedProduct header="Access" href="/cloudflare-one/" product="cloudflare-one">
+    Cloudflare Zero Trust replaces legacy security perimeters with Cloudflare's global network, making the Internet faster and safer for teams around the world
```
Comment (haleycode) [inline src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx:36]:
```suggestion
    Cloudflare Zero Trust replaces legacy security perimeters with Cloudflare's global network, making the Internet faster and safer for teams around the world.
```

**Review by haleycode — APPROVED** [review]
Some minor comments here for clarity and flow but approved!

File: src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (haleycode) [reply src/content/docs/reference-architecture/diagrams/sase/augment-access-with-serverless.mdx:90]: @bllchmbrs Thanks so much for reaching out about this! I'm the PM for the PCX team and I'll email you shortly to see if I can get you the info you're after.

**Pull request merged**